src/codegen/generate.ts

```diff
@@ -776,10 +776,35 @@ export default class ApiGenerator {
       return false;
     }
 
+    return this.checkSchemaOnlyModeHelper(
+      schema,
+      onlyMode,
+      resolveRefs,
+      new Set<string>(),
+    );
+  }
+
+  checkSchemaOnlyModeHelper(
```

Comment:
Do you think it's possible to merge `checkSchemaOnlyModeHelper` and `checkSchemaOnlyMode`? Or otherwise use a more clear name instead of the helper? Looks like something that should be internal API...

Reply:
We can define it as a nested function in `checkSchemaOnlyMode`. How do you think of this solution?

Reply:
Would prefer that, yeah